accounts/serializers.py

```diff
@@ -1,16 +1,36 @@
-import pytz
 from rest_framework import serializers
 
-from .models import Log
+from .models import Log, User
 
 
 class LogSerializer(serializers.HyperlinkedModelSerializer):
-    def validate_timezone(self, timezone_str):
-        if timezone_str not in pytz.all_timezones:
-            raise serializers.ValidationError(
-                f'Invalid timezone string {timezone_str}')
-        return timezone_str
-
     class Meta:
         model = Log
         fields = '__all__'
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    password = serializers.CharField(write_only=True)
+
+    def create(self, validated_data):
+        return User.objects.create_user(**validated_data)
```

Comment:
I believe you need to override the `create` method to save hashed password. Can you confirm this?

Reply:
Yes, we override create to save hashed password.

Reply:
kr dyn phir